hack/verify-generated-swagger-docs.sh

```diff
@@ -49,14 +49,14 @@ trap "cleanup" EXIT SIGINT
 
 cleanup
 
-mkdir -p "${_tmp}"
-cp -a -T "${DIFFROOT}" "${TMP_DIFFROOT}"
+mkdir -p "${TMP_DIFFROOT}"
+cp -a "${DIFFROOT}"/* "${TMP_DIFFROOT}"
```

Comment:
Does having a trailing slash on the destination directory make this more explicit?